MultiOracle.md

```diff
@@ -0,0 +1,462 @@
+There are two kinds of numeric outcome DLCs when it comes to using multiple oracles: those where
+there is no expectation or allowance for variance between oracles, and those where some small
+variance is acceptable and must be supported.
+
+For example, if there is some numeric data feed which broadcasts a single number daily, and multiple
+DLC oracles attest to this number, then all of these oracles should be expected to sign exactly the same
+number, down to the last binary digit.
+
+On the other hand if there is some less precise data source, such as a continuous price feed or multiple
+price feeds from different exchanges for the same asset, then oracles may differ by some small amount due
+to the non-deterministic nature of the data sampling as they may draw from the feed at slightly different times
+or from different exchanges entirely.
+In these kinds of DLCs, some amount of difference between oracles is expected and some larger difference
+is also acceptable.
+
+When executing the first kind of DLC, where no variance is allowed, then one should simply follow the same
+procedure as was [specified for enumerated outcomes above](#multiple-oracles-for-enumerated-outcome-events) because no macroscopic structure of the numeric
+nature of the outcomes is used from a multi-oracle support perspective.
+
+If instead the second kind of DLC is being executed, then a special, more intricate procedure detailed below
+must be used to ensure proper execution.
+This procedure differs from the simple enumerated or exact numeric case as it uses the numeric structure of
+related outcomes.
+Specifically distance is used so that outcomes are related if they are sufficiently near each other.
+
+### Design
+
+It turns out that there is an exceedingly large design space of approaches that can be taken to support multiple
+oracles with some acceptable variance.
+It seems to be generally true, however, that a trade-off exists between the precision of guarantees that a procedure
+can make, and the size of the multiplier on the number of adaptor signatures required for the DLC.
+This specification chooses to prioritize adaptor signature reduction at the expense of precise guarantees so as to more
+easily support more oracles, which should generally counteract some of the imprecision surrounding variance support.
+Although this resulting design is extremely opinionated in some sense, it does leave three parameters unspecified
+to be negotiated or chosen by DLC participants.
+
+The first two parameters are the orders of magnitude of expected and of allowed difference between oracles.
+By expected difference we mean the differences for which support is required by users, and by allowed difference
+we mean the differences for which support is acceptable but not required so that they may or may not be covered
+by the multi-oracle DLC construction algorithm.
+These parameters will, from now on, be called `minSupport` and `maxError` as they also represent the lower bound
```

Comment:
I'm confused by this part though I get the idea by looking on the illustration.

AFAICT, you have a primary interval to which you can superset a secondary interval thanks to `minSupport` bounds. But how do you decide on a primary interval as a DLC client ?

Further, should we speak about a tertiary interval that you can build thanks to `maxError` ?

Overall I think this part of the specification would benefit from a split in at least 3 sections: 
* "multi-oracle numeric range" data struct fields definition
* requirements on data struct as enforced by DLC counterparties
* design rational

I can take it latter if you wanna but I feel it's needed for this part of the spec to age well :)